go/src/koding/db/mongodb/modelhelper/modeltesthelper/modeltesthelper.go

```diff
@@ -37,7 +37,7 @@ func NewMongoDB(t *testing.T) *MongoDB {
 	}
 
 	if mongoURL == "" {
-		t.Skip("mongodb: one of env variables must be set:", strings.Join(mongoEnvs, ", "))
+		t.Fatalf("mongodb: one of env variables must be set: %s", strings.Join(mongoEnvs, ", "))
```

Comment:
@cihangir could you please update/remove comment [line 28](https://github.com/koding/koding/pull/9854/files#diff-83031beee3894bd6ef9bb2166959141eR28)